Sources/NostrSDK/Events/Calendars/CalendarEventParticipant.swift

```diff
@@ -54,6 +56,11 @@ public struct CalendarEventParticipant: PubkeyProviding, RelayProviding, Equatab
         self.tag = pubkeyTag
     }
 
+    /// Initializes a calendar event participant.
+    /// - Parameters:
+    ///   - pubkey: The public key of the participant.
+    ///   - relayURL: A relay in which the participant can be found
```

Comment:
Missing period at the end. For some reason GitHub wouldn't let me do a suggestion here.

Reply:
Fixed!